server/controllers/column.js

```diff
@@ -0,0 +1,53 @@
+const asyncHandler = require("express-async-handler");
+const Column = require("../models/Column");
+
+exports.getColumns = asyncHandler(async (req, res) => {
+  const columns = await Column.find();
```

Comment:
This would get every single users columns even ones that aren't related to a users board. Add a filter to make sure we are only grabbing the users columns.

Reply:
Makes more sense to filter just by the board ID but if we attach a user to columns either one works.

Reply:
Won't be fetching columns here; will just populate with the board as we respond from the backend